### PR TITLE
Put drops in inventory

### LINKED ISSUE #889: Survival mode enhancement
When someone is in survival mode and they break a block, instead of the item dropping on the ground (and thus being auto-removed) it would go directly into their inventory. Why this would be useful: ops could set up and play things like Skyblock on the server a lot easier. As it is currently, survival mode play on the server is not practical because of how TFM handles drops, and this would improve that quite a bit without introducing entity lag issues.

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Resolves #889

### CODE REVIEW

**Comment by LegendIsAwesomes** [comment]
@JeromSar I had to make a new PR as I had no choice considering my last one wasn't made in the github website application, I have now closed the other PR.

File: nbproject/project.properties [hunk 1]
```diff
@@ -43,6 +43,10 @@ dist.jar=${dist.dir}/TotalFreedomMod.jar
 dist.javadoc.dir=${dist.dir}/javadoc
 endorsed.classpath=
 excludes=
+file.reference.BukkitTelnet.jar=C:\\Users\\ann\\Desktop\\Coding stuff\\BukkitTelnet.jar
```
Comment (rsajeey) [inline nbproject/project.properties]: Why is this here?

File: buildnumber.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #Build Number for ANT. Do not edit!
-#Sun Sep 06 17:00:40 CEST 2015
-build.number=1055
+#Fri Dec 25 21:04:42 GMT 2015
+build.number=1059
```
Comment (rsajeey) [inline buildnumber.properties]: You're not allowed to change the buildNumber.

File: nbproject/project.properties [hunk 3]
```diff
@@ -51,7 +55,11 @@ javac.classpath=\
     ${libs.SpigotServer.classpath}:\
     ${libs.TF-Essentials.classpath}:\
     ${libs.TF-WorldEdit.classpath}:\
-    ${libs.BukkitTelnet.classpath}
+    ${libs.BukkitTelnet.classpath}:\
```
Comment (rsajeey) [inline nbproject/project.properties]: Why is this here?

File: nbproject/project.properties [hunk 4]
```diff
@@ -51,7 +55,11 @@ javac.classpath=\
     ${libs.SpigotServer.classpath}:\
     ${libs.TF-Essentials.classpath}:\
     ${libs.TF-WorldEdit.classpath}:\
-    ${libs.BukkitTelnet.classpath}
+    ${libs.BukkitTelnet.classpath}:\
+    ${file.reference.BukkitTelnet.jar}:\
```
Comment (rsajeey) [inline nbproject/project.properties]: Why is this here?

**Comment by LegendIsAwesomes** [comment]
Well, it somehow decided to do it itself ;-;

**Comment by xDestroyer217** [comment]
Might want to remove it before it's pushed, @LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
Ye, good idea, atm not on my PC I'm on mobile, but as soon as I get on my PC I will.

File: buildnumber.properties (hunk 2 above)
Comment (JeromSar) [inline buildnumber.properties]: @rsajeey Yes you are.

File: nbproject/project.properties (hunk 1 above)
Comment (JeromSar) [inline nbproject/project.properties]: @LegendIsAwesomes Please uncommit all changes made to this file. You probably should learn how NetBeans' library function works.

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java [hunk 5]
```diff
@@ -111,6 +112,14 @@ else if (playerLocation.distanceSquared(location) > (nukeMonitorRange * nukeMoni
                 }
             }
         }
+        if (player.getGameMode().equals(GameMode.SURVIVAL))
+        {
+            for (ItemStack stack : event.getBlock().getDrops())
+            {
+                event.getBlock();
+                player.getInventory().addItem(stack);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java]: This does not take into account if entity wiper is enabled or not. If it isn't, this might result in duplicate drops.

**Comment by LegendIsAwesomes** [comment]
@JeromSar mind pointing out what I need to fix or remove.

**Comment by LegendIsAwesomes** [comment]
@JeromSar I took notice of the entity wiper myself, I chose to ignore and test it, it does not drop multiple entities.

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java (hunk 5 above)
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java]: Additionally, this should probably be in a separate event with `ignoreCancelled = true` and `priority = EventPriority.MONITOR` to only drop items if the event was not cancelled.

**Comment by JeromSar** [comment]
@LegendIsAwesomes TFM's entity wiper is enabled by default. I have the feeling that if it's disabled, duplicate drops will occur. For other changes, please see the other comments on this PR. :)

**Comment by LegendIsAwesomes** [comment]
Okay, is it config related?

**Comment by JeromSar** [comment]
@LegendIsAwesomes Yes, it's default state is enabled and disabled through the config.

**Comment by LegendIsAwesomes** [comment]
What config is this found in, it's not in the config.yml

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java [hunk 6]
```diff
@@ -111,6 +112,14 @@ else if (playerLocation.distanceSquared(location) > (nukeMonitorRange * nukeMoni
                 }
             }
         }
+        if (player.getGameMode().equals(GameMode.SURVIVAL))
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java]: Change this to:

`         if (player.getGameMode().equals(GameMode.SURVIVAL))
         {
          if (TFM_ConfigEntry.AUTO_ENTITY_WIPE.getBoolean()) 
             {
                 if (TFM_ConfigEntry.AUTO_ENTITY_WIPE.equals(false)) 
                 {
                     return;
                 }
                 else
                 {
             for (ItemStack stack : event.getBlock().getDrops())
             {
                 event.getBlock();
                 player.getInventory().addItem(stack);
             }
            }
                 
             }
         }`

**Comment by LegendIsAwesomes** [comment]
I'm blind me ;-; I just found it while messing about. And the formatting is a bit bad so ima do it myself.

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java (hunk 5 above)
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java]: @JeromSar, I have maybe fixed this in my post.

**Comment by DUFCLiam** [comment]
@LegendIsAwesomes, I have made changes you can edit the format if you would like to.

**Comment by LegendIsAwesomes** [comment]
I am most definitely doing that.
